.github/workflows/publish-php.yml

```diff
@@ -0,0 +1,75 @@
+name: Publish PHP SDK
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build_rust:
+    uses: ./.github/workflows/build-rust-cross-platform.yml
+
+  setup_php:
+    name: Setup PHP
+    runs-on: ubuntu-22.04
+    needs:
+      - build_rust
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Setup PHP with PECL extension
+        uses: shivammathur/setup-php@7fdd3ece872ec7ec4c098ae5ab7637d5e0a96067 # 2.26.0
+        with:
+          php-version: '8.0'
+          tools: composer
+          extensions: ext-ffi
+
+      - name: Composer check
+        run: |
+          composer install
+          composer validate
+        working-directory: languages/php/
+
+      - name: Download x86_64-apple-darwin files
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: libbitwarden_c_files-x86_64-apple-darwin
+          path: temp/macos-x64
+
+      - name: Download aarch64-apple-darwin files
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: libbitwarden_c_files-aarch64-apple-darwin
+          path: temp/macos-arm64
+
+      - name: Download x86_64-unknown-linux-gnu files
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: libbitwarden_c_files-x86_64-unknown-linux-gnu
+          path: temp/ubuntu-x64
+
+      - name: Download x86_64-pc-windows-msvc files
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: libbitwarden_c_files-x86_64-pc-windows-msvc
+          path: temp/windows-x64
+
+      - name: Copy lib files
+        run: |
+          mkdir -p languages/php/src/lib/macos-arm64
+          mkdir -p languages/php/src/lib/ubuntu-x64
+          mkdir -p languages/php/src/lib/macos-x64
+          mkdir -p languages/php/src/lib/windows-x64
+
+          platforms=("macos-arm64" "ubuntu-x64" "macos-x64" "windows-x64")
+          files=("libbitwarden_c.dylib" "libbitwarden_c.so" "libbitwarden_c.dylib" "bitwarden_c.dll")
+
+          for ((i=0; i<${#platforms[@]}; i++)); do
+            cp "temp/${platforms[$i]}/${files[$i]}" "languages/php/src/lib/${platforms[$i]}/${files[$i]}"
+          done
+
+      - name: Publish version
+        run: curl -XPOST -H'content-type:application/json' 'https://packagist.org/api/update-package?username=malirobot&apiToken=${{secrets.PACKAGIST_KEY}}' -d'{"repository":{"url":"https://packagist.org/packages/bitwarden/sdk"}}'
```

Comment:
@mimartin12 Should this be updated with some Bitwarden managed credentials, or do we leave that for after the merge?

Reply:
What we can do is that I can add someone from Bitwarden as a collaborator on packagist, then the access to the key won't be a problem (it is used in action secrets). The other scenario is to make a completely new package, the only potential problem, one that I haven't thought about before, is that the name of the package is reserved to "bitwarden/sdk", so it would need to be named differently.

Reply:
What @Hinton is suggesting sounds like a good way to go about it, especially since you will want to preserve the package name. I am not familiar with Packagist, but it looks like their [API tokens](https://packagist.org/about) are scoped to the user account. That means `${{secrets.PACKAGIST_KEY}}` will need to be updated when you do the switch.
![image](https://github.com/bitwarden/sdk/assets/77340197/69b76867-4e93-4c36-9782-efdfaaf435b5)